modules/nw-route-specific-annotations.adoc

```diff
@@ -56,6 +57,37 @@ Setting a server-side timeout value for passthrough routes too low can cause
 WebSocket connections to timeout frequently on that route.
 ====
 
+.A route that allows only one specific IP address
+[source,yaml]
+----
+metadata:
+  annotations:
+    haproxy.router.openshift.io/ip_whitelist: 192.168.1.10
+----
+
+.A route that allows several IP addresses
+[source,yaml]
+----
+metadata:
+  annotations:
+    haproxy.router.openshift.io/ip_whitelist: 192.168.1.10 192.168.1.11 192.168.1.12
+----
+
+.A route that allows an IP address CIDR network
+[source,yaml]
+----
+metadata:
+  annotations:
+    haproxy.router.openshift.io/ip_whitelist: 192.168.1.0/24
+----
+
+.A route that allows both IP addresses and IP address CIDR networks
```

Comment:
This is a small nit, but in the example, there is only one IP address. The title says "IP addresses." Could you change the title to "A route that allows both an IP address and IP address CIDR networks?" Or keep the title the same and add a second IP address to the example?